ui.R

```diff
@@ -20,6 +20,8 @@ ui = shiny::navbarPage(
                           primary = "#003E74",
                           secondary = "#9D9D9D"),
 
+  shinyjs::useShinyjs(),
```

Comment:
I'm getting a 

> Warning: Navigation containers expect a collection of `bslib::nav()`/`shiny::tabPanel()`s and/or `bslib::nav_menu()`/`shiny::navbarMenu()`s. Consider using `header` or `footer` if you wish to place content above (or below) every panel's contents.

warning that wasn't there before. If we wrap the shinyjs call inside a tagList, it should fix this: e.g. ui = shiny::tagList(shinyjs::useShinyjs(), shiny::navbarPage(...))